setup_sklearnex.py

```diff
@@ -89,8 +89,13 @@
 if build_distribute:
     packages_with_tests += [
         'sklearnex.spmd',
-        'sklearnex.spmd.ensemble',
-        'sklearnex.spmd.linear_model']
+        'sklearnex.spmd.decomposition']
+    if ONEDAL_VERSION >= 20230100:
+        packages_with_tests += [
+            'sklearnex.spmd.basic_statistics',
+            'sklearnex.spmd.ensemble',
+            'sklearnex.spmd.linear_model',
+            'sklearnex.spmd.neighbors']
```

Comment:
Could be some master merge artefact. `sklearnex.spmd.ensemble` is not depend on ONEDAL_VERSION >= 20230100.
```suggestion
        'sklearnex.spmd.decomposition',
        'sklearnex.spmd.ensemble']
    if ONEDAL_VERSION >= 20230100:
        packages_with_tests += [
            'sklearnex.spmd.basic_statistics',
            'sklearnex.spmd.linear_model',
            'sklearnex.spmd.neighbors']
```